src/Entity/Adherent.php

```diff
@@ -157,6 +158,13 @@ class Adherent implements UserInterface, GeoPointInterface, EncoderAwareInterfac
      */
     private $memberships;
 
+    /**
+     * @var Summary|null
+     *
+     * @ORM\OneToOne(targetEntity="Summary", mappedBy="member", cascade={"all"})
```

Comment:
This is not the owner side, will test it later if it's worth it but it should not be needed.